### PR TITLE
Mob 3350 part 2

### PR DESCRIPTION
**Pull request by cosminBadulescu**
## JIRA
[MOB-3350](https://karhoo.atlassian.net/browse/MOB-3350)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [x] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt [hunk 1]
```diff
@@ -139,24 +123,27 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
     fun onResume() {
         isGuest = isGuest()
-        presenter.setBookingEnablement(bookingRequestPassengerDetailsWidget.allFieldsValid())
     }
 
     @OnLifecycleEvent(Lifecycle.Event.ON_STOP)
     fun onStop() {
-        disableBooking()
+//        disableBooking()
```
Comment (mohadian) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt:130]: we can remove the commented code?

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt [hunk 2]
```diff
@@ -404,17 +393,13 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
     }
 
     override fun handlePaymentDetailsUpdate() {
-        presenter.setBookingEnablement(bookingRequestPassengerDetailsWidget.allFieldsValid())
+        // not used
```
Comment (mohadian) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt:396]: Shall we remove the handlePaymentDetailsUpdate if it is no longer in use?

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt [hunk 3]
```diff
@@ -139,24 +123,27 @@ class BookingRequestView @JvmOverloads constructor(context: Context,
...
         cancelButton.setOnClickListener {
-            disableBooking()
             presenter.clearData()
             hideSoftKeyboard()
             hideWindow()
         }
 
+        bookingRequestLinearLayout.setOnClickListener {
+            it.hideSoftKeyboard()
+            Toast.makeText(context, "Click", Toast.LENGTH_SHORT).show()
```
Comment (mohadian) [inline uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt:144]: Please remove the toast :)

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt (hunk 2 above)
Comment (cosminBadulescu) [reply uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt:396]: cannot do it, this interface is used in other places and it has multiple methods. We only do not use that one

File: uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt (hunk 1 above)
Comment (cosminBadulescu) [reply uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestView.kt:130]: removed

**Comment by cosminBadulescu** [comment]
Added new commits but it isn't refreshing, closing PR and opening again

**Pull request closed without merging**